tools/validate_configs/test_configs/two-clusters-sql.yaml

```diff
@@ -0,0 +1,155 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blueprint_name: two-clusters
+
+vars:
+  project_id: hpc-toolkit-demo
+  deployment_name: clstr2
+  region: us-east1
+  zone: us-east1-b
+  enable_reconfigure: True
+  enable_cleanup_compute: False
+  enable_cleanup_subscriptions: True
+  enable_bigquery_load: True
+  instance_image_custom: True
+
+deployment_groups:
+- group: net
+  modules:
+  - source: modules/network/vpc
+    id: hpc_network
+
+  - source: ./community/modules/network/private-service-access
+    id: ps-connect
```

Comment:
Please remove dash in favor of underscore

Reply:
you mean the ids, right?

Reply:
done